enclave-runtime/src/tls_ra/tls_ra_client.rs

```diff
@@ -129,13 +129,17 @@ where
 	}
 
 	/// Reads the payload header, indicating the sent payload length and type.
-	fn read_header(&mut self, start_bytes: Vec<u8>) -> EnclaveResult<TcpHeader> {
+	fn read_header(&mut self, start_byte: u8) -> EnclaveResult<TcpHeader> {
+		const PAYLOAD_SIZE_LENGTH: usize = 8; // u64 indicating the payload size is 8 bytes
```

Comment:
```suggestion
		const PAYLOAD_SIZE_IN_BYTES: usize = 8;
```

The comment is already not correct anymore (no u64), so maybe we can get rid of it completely?

Reply:
The comment is still correct, but maybe confusing. The constant is the byte size of an `u64`, which is 8 bytes.